fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java

```diff
@@ -1075,6 +1078,24 @@ public static long analyzeRowStorePageSize(Map<String, String> properties) throw
         return rowStorePageSize;
     }
 
+    public static long analyzeStoragePageSize(Map<String, String> properties) throws AnalysisException {
```

Comment:
Why do we need this procedure depends on a constants which may differ from BE side 
`STORAGE_PAGE_SIZE_DEFAULT_VALUE`? we have to maintain both side which is dangerous and easy to forget to keep them consistent.

The idea situation is that the missing or default value is processed  on BE
 side. 
* missing -> 64KB, this is also the compatibility of the data written before this PR we should take care of.
* default -> 64KB

FE does not have to care about the absence of filed of `storage_page_size`